x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/StaticPlot.js

```diff
@@ -138,21 +139,53 @@ class StaticPlot extends PureComponent {
     }
   }
 
+  /**
+   * A tick format function that takes the timezone from Kibana's settings into
+   * account. Used if no tickFormatX prop is supplied.
+   *
+   * This produces the same results as the built-in formatter from D3, which is
+   * what react-vis uses, but shifts the timezone.
+   */
+  tickFormatX = value => {
```

Comment:
Should this perhaps be named something like `tickFormatXTime` to indicate it's not a generic formatter but one specifically for formatting time values?

Reply:
Changed in b78868ae249.